alica_engine/src/engine/RuleBook.cpp

```diff
@@ -228,6 +229,8 @@ PlanChange RuleBook::authorityOverrideRule(RunningPlan& r)
  */
 PlanChange RuleBook::planAbortRule(RunningPlan& r)
 {
+    if (!_autoFailureHandlingEnabled)
+        return PlanChange::NoChange;
     assert(!r.isRetired());
```

Comment:
my bad, I didn't want to disable the planAbortRule but forgot to remove this piece of code. The rule just marks the running plan as failure but does not handle it in any way. The only thing it seems to do is return PlanChange::Failure, so that the higher level plan can react to the failure in the same tick() of the running plan, so I think its ok to not disable the rule.

Reply:
I did enable this rule & the tests still work as expected